stix/info.json

```diff
@@ -149,160 +148,57 @@
-                "visible": true,
-                "type": "text",
-                "name": "rec_class",
-                "placeholder": "organization",
-                "tooltip": "The type of entity that Receiving Identity describes, e.g., an individual or organization.",
-                "description": "The type of entity that Receiving Identity describes, e.g., an individual or organization."
-              },
-              {
-                "title": "Description",
-                "required": false,
-                "editable": true,
-                "visible": true,
-                "type": "text",
-                "name": "rec_org_description",
-                "description": "A description that provides more details and context about the Receiving Identity, potentially including its purpose and its key characteristics."
-              },
-              {
-                "title": "Role",
-                "required": false,
-                "editable": true,
-                "visible": true,
+                "title": "Filename",
+                "name": "filename",
                 "type": "text",
-                "name": "rec_role",
-                "placeholder": "Cyber Security",
-                "tooltip": "The list of roles that Receiving Identity performs (e.g., CEO, Domain Administrators, Doctors, Hospital, or Retailer).",
-                "description": "The list of roles that Receiving Identity performs (e.g., CEO, Domain Administrators, Doctors, Hospital, or Retailer). "
-              },
-              {
-                "title": "Sector",
-                "required": false,
-                "editable": true,
                 "visible": true,
-                "type": "text",
-                "name": "rec_sector",
-                "placeholder": "technology",
-                "tooltip": "The industry sectors that Receiving Identity belongs to.",
-                "description": "The industry sectors that Receiving Identity belongs to."
-              },
-              {
-                "title": "Email ID",
-                "required": false,
                 "editable": true,
-                "visible": true,
-                "type": "text",
-                "name": "rec_contact_info",
-                "tooltip": "The contact information (e-mail, phone number, etc.) for Receiving Identity.",
-                "description": "The contact information (e-mail, phone number, etc.) for Receiving Identity."
+                "required": true,
+                "placeholder": "Eg. demo.json",
+                "tooltip": "The file will be created under /tmp"
               }
             ]
           },
-          "name": "rec_spec_info",
-          "tooltip": "Add details of Identity who is intended to receive the threat information ",
-          "description": "Add details of Identity who is intended to receive the threat information "
+          "tooltip": "If enabled, the feed will be written to a value, else returned as a JSON",
```

Comment:
If enabled, the feed will be written to a file with this name, else returned as a JSON